.pre-commit-config.yaml

```diff
@@ -0,0 +1,46 @@
+repos:
+  - repo: https://github.com/PyCQA/autoflake
+    rev: v1.7.7
+    hooks:
+      - id: autoflake
+        args:
+          - --in-place
+          - --remove-duplicate-keys
+  - repo: https://github.com/PyCQA/isort
+    rev: 5.10.1
+    hooks:
+      - id: isort
+  - repo: https://github.com/PyCQA/flake8
+    rev: 6.0.0
+    hooks:
+      - id: flake8
+        additional_dependencies: &flake8_dependencies
+          - flake8-annotations
+          - flake8-bugbear
+          - flake8-docstrings
+          - flake8-isort
```

Comment:
is there a reason to have **both** `isort` (introduced through the new pre-hooks) and `flake8-isort` (was previously used as the `isort` wrapper)?

_could_ theoretically cause an issue that `isort` is different version of `flake8-isort` and thus they keep wrangling imports to their desired order to infinity (not a huge risk as `isort` is quite stable)

Reply:
Oh, yeah, good copypaste catch.